### ISSUE TITLE
status bar project name not displayed if path like "D:/ABC"

### ISSUE THREAD

**Issue by ss1969** [fabiospampinato/vscode-projects-plus#17]
status bar project name not displayed if path like "D:/ABC".
D must be d, and / must be \\\\
or the status bar will not display the project name.

**Comment by fabiospampinato** [comment]
Can you share the exact entry in your settings that's causing problems?

**Comment by jemhuntr** [comment]
```
            {
              "name": "Test Project",
              "description": "Some description.",
              "path": "C:\\Users\\username\\Documents\\Projects\\test-project"
            },
```

Having trouble getting the names to show on the status bar as well. It just shows "No project" no matter which project I open.

**Comment by ss1969** [comment]
` {
              "name": "Test Project",
              "description": "Some description.",
              "path": "C:\\Users\\username\\Documents\\Projects\\test-project"
            },

`

in this configuration, 'C' must be changed to 'c', or "No project" is shown;
and if we change "\\" to '/', there will be this bug also.

**Comment by fabiospampinato** [comment]
This should be fixed in v1.15.1.
